### ISSUE TITLE
su2Deform python 3.7 error

### ISSUE THREAD

**Issue by galbramc** [su2code/SU2#620]
I get the following error from su2Deform when using python 3.7 and SU2 6.1.0:

```
su2Deform( "testing.cfg", args.numberProc) # Run SU2 mesh deformation
  File "$SU2_RUN/mesh_deformation.py", line 83, in mesh_deformation
    info = SU2.run.DEF(config)
  File "$SU2_RUN/SU2/run/interface.py", line 158, in DEF
    konfig.dump(tempname) 
  File "$SU2_RUN/SU2/io/config.py", line 125, in dump
    dump_config(filename,self)
  File "$SU2_RUN/SU2/io/config.py", line 961, in dump_config
    write_config(filename,config)    
  File "$SU2_RUN/SU2/io/config.py", line 775, in write_config
    n_lists = len(new_value)
TypeError: object of type 'map' has no len()
```

Any help fixing this would be greatly appreciated.

**Comment by galbramc** [comment]
@economon it does work with python 2.7, I have not tried multiple versions of python 3, just 3.7. The failure occurs with the python script and does not get to the point where it starts executing SU2_DEG. The minimum example to reproduce the error is:

```
from mesh_deformation import mesh_deformation as su2Deform
su2Deform( "testing.cfg" )
```
using [testing.cfg](https://github.com/su2code/SU2/files/2635991/testing.txt), but with a txt extension.

**Comment by rsanfer** [comment]
Hi all, I am also seeing this error with Python 3.7.2 in the finite_difference.py script. I replaced 'len(new_value)' with 'len(list(new_value))' here and there but then the DV_VALUE_NEW doesn't get correctly set. Any new hints?

**Comment by petebachant** [comment]
@rsanfer can you post the traceback?

**Comment by talbring** [comment]
I fixed that bug in #641. (https://github.com/su2code/SU2/pull/641/commits/c22a5c5cfe7daec4deacbec621d278d26362c5d3)
